compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DataflowAnalyzer.java

```diff
@@ -357,6 +357,14 @@ private void checkForUninitializedGlobalVars(List<BLangVariable> globalVars) {
                 case TUPLE_VARIABLE:
                     checkForUninitializedGlobalVars(((BLangTupleVariable) globalVar).memberVariables);
                     break;
+                case RECORD_VARIABLE:
+                    BLangRecordVariable recordVariable = (BLangRecordVariable) globalVar;
+                    List<BLangVariable> memberVariables = new ArrayList<>();
+                    recordVariable.variableList.forEach(memberKeyValue -> {
+                        memberVariables.add(memberKeyValue.valueBindingPattern);
+                    });
```

Comment:
Let's replace this with a normal for loop.

Reply:
Are we always removing forEach loops?. I used it here because it's just one line of body.